### DISCUSSION TITLE
Forked GitHub Classroom assignments NO longer show on contributions graph

### DISCUSSION THREAD

**Discussion by Web-Dev-Codi** [community/community#136306] [GitHub Education] [answered]
### Hi everyone,

After July 2nd, I started encountering a 500 Internal Server Error, which I reported.  [Report](https://github.com/orgs/community/discussions/130738#discussion-6889223)

Around the same time, GitHub Classroom assignments transitioned to a forked approach. Since then, I've noticed that my commits and pushes are not being reflected on my contributions graph on my main account.

Is there anything I can do to resolve this? I've been dedicating significant time to my school projects, and for developers, those green contribution squares are a badge of honor. It would be disappointing not to be able to showcase my hard work and passion for coding.

**Comment by gpincheiraa** [comment]
very interested in this topic.
I ve been teaching in a course around 2 months, the students works in 15 or more assigments, but commits are not reflected in their contribution graphs.

**Comment by Thesirix** [comment, marked as answer]
Here are a few steps you can try to resolve this issue:

**Check if your commits are properly linked to your GitHub account**: Make sure that the email address used in your commits is associated with your GitHub account. If the email in your commits doesn't match your GitHub account, the contributions won't appear on your graph.

**Ensure the repository isn't private**: Contributions in private repositories don't appear on the contribution graph unless you enable the option to display your private contributions in your GitHub account settings.

**Merge your contributions into the main repository**: If you're working on forks, make sure that your commits are merged into the main repository. Contributions in forks might not appear on your contribution graph unless they are merged into the main branch.

**Be patient**: Sometimes there can be a delay before the contribution graph updates after changes are made in repositories.

If these steps don't solve the issue, I recommend reaching out to GitHub support with these details.

**Reply by Web-Dev-Codi** [reply]
Since it's a school organization, the repos are private. With that said I eventually figured out that as you said, clone , create a working branch, push, merge to main. Now I can see the contributions graph update which is nice. 

Also, there should have been a more official release memo to the organization leaders detailing the proper steps students should make. 

I have been working/studying countless hours that will never be revealed on my graph. I realize I am to attached to that green squares, but hey, we are developers and that's how most of use are. 

Further more, the school organization will eventually be deleted, which will erase the contributions. So I'm going create a private repo to reflect my work which will keep the contribution graph intact, and also act as a reference for topics learned.

I am a little more relaxed about it because of the work around I mentioned earlier. 

Let's hope there comes a time when contributions are permeant regardless of repo deletion.

**Comment by ryan-winkler** [comment]
@Web-Dev-Codi @Thesirix @gpincheiraa @mgriffin @martinwoodward  I've opened a feature request for this to be changed.

https://github.com/orgs/community/discussions/205090

If you can comment how substantial work is being hidden due to this rule, It will help strengthen the case
